python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -65,6 +65,9 @@ class ImageDifferenceConfig(pexConfig.Config):
         doc="Use a simple gaussian PSF model for pre-convolution (else use fit PSF)? "
             "Ignored if doPreConvolve false.")
     doDetection = pexConfig.Field(dtype=bool, default=True, doc="Detect sources?")
+    doDecorrelation = pexConfig.Field(dtype=bool, default=False,
+        doc="Perform diffim decorrelation correction similar to Kaiser? If True, also updates the "
```

Comment:
OK, changed to "Perform diffim decorrelation to undo pixel correlation due to convolution?"